### PR TITLE
Add isEmpty documentation file for Swift Arrays

### PR DESCRIPTION
**Pull request by noahtq**
### Description

- Adding new file for documentation of the Swift .isEmpty array method
- Closes Issue #2630 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (dakshdeepHERE) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:33]: why don't you create a output section for the outputs. Something like this
image

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Suggestion from my side

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (noahtq) [reply content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:33]: I will make this change. Thank you for the feedback!

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:15]:
```suggestion
The **`.isEmpty`** property returns true if an array contains no elements, returns false if the array contains one or more elements.
```

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:21]: There should be a sentence after the syntax block explaining what `arrayName` is.

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (SSwiniarski) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:24]: There should be a sentence before the example summarizing what it is doing.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@noahtq I did a first maintainer review and left a few comments. Mostly things I'd like to see added to the entry.

**Comment by SSwiniarski** [comment]
@noahtq Are you stll working on this entry? Do you need assistance implementing my comments?

**Review by SSwiniarski — APPROVED** [review]
@noahtq To get this moving I made the requested edits myself. Now I'm passing this on to a second review.

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:15]:
```suggestion
The **`.isEmpty`** property returns `true` if an array contains no elements, and returns `false` if the array contains one or more elements.
```

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:23]:
```suggestion
Where `arrayName` is the array being checked for elements.
```

File: content/swift/concepts/arrays/terms/isEmpty/isEmpty.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/isEmpty/isEmpty.md:27]:
```suggestion
The following example creates and array and evaluates the contents of the array with `.isEmpty` before and after adding an element:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@noahtq I've added a few additional comments. Please update the entry, and tag me when you've finished the changes. Thanks

**Comment by SSwiniarski** [comment]
@caupolicandiaz I implemented your suggestions. Does this look ready to merge now?

**Review by caupolicandiaz — APPROVED** [review]
@noahtq @SSwiniarski Looks good

**Pull request merged**
